### PR TITLE
Add Alipay, Baidu, Bilibili and Zhihu websites:

### PR DESCRIPTION
**Pull request by tomac4t**
- Add Alipay as medium
- Add Baidu as medium
- Add Bilibili as medium
- Change QQ as medium
- Add notes for QQ
- Update url for QQ (The former is Wechat one, not QQ.)
- Change Wechat as medium
- Add notes for Wechat
- Add Zhihu as easy
- Add Weibo as medium
- Add Elsevier.com as hard

### CODE REVIEW

**Review by WebSnke — CHANGES_REQUESTED** [review]
I added some suggestions to your code.

**Comment by WebSnke** [comment]
It would be great if you could create separate pull requests in the future.

File: _data/sites.json [hunk 1]
```diff
@@ -5173,6 +5201,16 @@
         ]
     },
 
+    {
+        "name": "Elsevier",
+        "url": "https://service.elsevier.com/app/answers/detail/a_id/30148/supporthub/ecommerce/~/how-do-i-request-deletion-of-my-elsevier.com-account",
```
[the remark this answers is not in the document]
Comment (WebSnke) [reply _data/sites.json:5206]: > I don't see the support email on that page:(

Yeah, me neither.

File: _data/sites.json [hunk 2]
```diff
@@ -1947,6 +1966,15 @@
...
 
+    {
+        "name": "Bilibili",
+        "url": "https://passport.bilibili.com/register/logout.html",
+        "difficulty": "medium",
+        "domains": [
+            "bilibili.com"
+        ]
+    },
+
```
Comment (tupaschoal) [inline _data/sites.json:1977]: Needs to be moved after Bigstock to fix alphabetical sorting.

File: _data/sites.json [hunk 3]
```diff
@@ -781,6 +781,16 @@
         ]
     },
 
+    {
+        "name": "AliPay",
+        "url": "https://memberprod.alipay.com/account/close/apply.htm",
+        "difficulty": "easy",
+        "notes": "From the app: Settings → Account and Security → Account Cancel.",
```
Comment (tupaschoal) [inline _data/sites.json:788]:
```suggestion
        "notes": "From the app: Settings ⮕ Account and Security ⮕ Account Cancel.",
```

File: _data/sites.json [hunk 4]
```diff
@@ -1653,6 +1663,15 @@
         ]
     },
 
+    {
+        "name": "Baidu",
+        "url": "https://passport.baidu.com/v6/cancelAccount",
+        "difficulty": "medium",
```
Comment (tupaschoal) [inline _data/sites.json:1669]: Why is it medium? I feel like this needs some explanation

File: _data/sites.json [hunk 5]
```diff
@@ -1958,6 +1977,15 @@
         ]
     },
 
+    {
+        "name": "Bilibili",
+        "url": "https://passport.bilibili.com/register/logout.html",
+        "difficulty": "medium",
```
Comment (tupaschoal) [inline _data/sites.json:1983]: Same comment about being medium here.

File: _data/sites.json [hunk 6]
```diff
@@ -13721,8 +13759,9 @@
 
     {
         "name": "QQ",
-        "url": "https://support.weixin.qq.com/security/readtemplate?page=statement_page&t=account_cancellation/w_index&wechat_real_lang=en_US",
+        "url": "https://kf.qq.com/faq/1803146n2yMr180314MbIfER.html",
         "difficulty": "easy",
+        "notes": "From the app: Avatar → Settings → Account Security → Account Cancellation.",
```
Comment (tupaschoal) [inline _data/sites.json:13764]:
```suggestion
        "notes": "From the app: Avatar ⮕ Settings ⮕ Account Security ⮕ Account Cancellation.",
```

File: _data/sites.json [hunk 7]
```diff
@@ -18265,9 +18304,11 @@
         "name": "WeChat",
         "url": "https://help.wechat.com/cgi-bin/micromsg-bin/oshelpcenter?opcode=2&id=1706083AnYFb170608VF3Ur2",
         "difficulty": "easy",
+        "notes": "From the app: Me → Settings → Account Security → WeChat Security Center → Account Cancellation.",
```
Comment (tupaschoal) [inline _data/sites.json:18307]:
```suggestion
        "notes": "From the app: Me ⮕ Settings ⮕ Account Security ⮕ WeChat Security Center ⮕ Account Cancellation.",
```

**Review by tupaschoal — COMMENTED** [review]
Most of the comments are just to standardize the arrows we just changed, but there are a few questions there.

File: _data/sites.json (hunk 4 above)
Comment (tupaschoal) [reply _data/sites.json:1669]: I see

**Pull request merged**